src/openfermion/ops/_code_operator_test.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from openfermion.ops import SymbolicBinary
+from openfermion.ops._code_operator import BinaryCodeError, \
+    BinaryCode, linearize_decoder
+
+
+class BinaryCodeTest(unittest.TestCase):
```

Comment:
I am virtually certain pep8 linter will complain about the lack of spaces here. Please remember to run pep8 linter: https://pypi.python.org/pypi/pep8